frontend/src/index.tsx

```diff
@@ -55,7 +55,15 @@ const App: React.FC<PropsWithChildren> = ({ children }) => {
 const AppRoutes = () =>
   useRoutes([
     { path: "/", element: <LandingPage /> },
-    { path: "/admin/*", element: <ManageAdmissions /> },
+    {
+      path: "/admin/*",
+      element:
+        djangoData.user.is_staff || djangoData.user.is_member_of_webkom ? (
+          <ManageAdmissions />
+        ) : (
+          <p>Du har ikke tilgang til denne siden.</p>
+        ),
+    },
```

Comment:
Best practice here is to move the logic to a component wrapping `ManageAdmissions` for consistency

```js
<RequireAuth auth={djangoData.user.is_staff || djangoData.user.is_member_of_webkom}>
    <ManageAdmissions />
<RequireAuth />
```

same goes for all other places it is used :)